index.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+session_start();
+
+if (!$_SESSION['loggedIn']) {
+    header('Location:login.php');
+}
+
+require_once ('vendor/autoload.php');
+
+use Pokedex\DbConnection;
+use Pokedex\PokeList;
+
+$db = new DbConnection();
+$pokeList = new PokeList($db->getDB(), $_SESSION['userID']);
```

Comment:
This is a warning more than a requested change: this will probably cause the mother of all merge conflicts. I think that pokeStatus from mine should be updated from the $_POST data before the pokedata is retrieved from the db.

Reply:
Thanks for pointing out will keep in mind when merging